firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -236,6 +237,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
                 if value {
                     self?.gleanLifecycleObserver.startObserving()
                 } else {
+                    GleanMetrics.Pings.shared.usageDeletionRequest.submit()
```

Comment:
This is a `follows_collection_enabled=false` ping, thus it needs to be specifically enabled, otherwise it won't ever be submitted.
Because it also should carry the usage_profile_id it needs to be enabled before that metric is set so that we actually store that ID for it.